src/views/ModelPlan/TaskList/Basics/Milestones/index.tsx

```diff
@@ -0,0 +1,634 @@
+
+                    <FieldGroup
+                      scrollElement="phasedIn"
+                      error={!!flatErrors.phasedIn}
+                      className="margin-top-4"
+                    >
+                      <Label htmlFor="phasedIn">{t('tightTimeline')}</Label>
+                      <span className="usa-hint display-block text-normal margin-top-1">
+                        {t('tightTimelineInfo')}
+                      </span>
+                      <FieldErrorMsg>{flatErrors.phasedIn}</FieldErrorMsg>
+                      <Fieldset>
+                        <Field
+                          as={Radio}
+                          id="phasedIn-Yes"
+                          name="phasedIn"
+                          label={h('yes')}
+                          value="YES"
+                          checked={values.phasedIn === true}
+                          onChange={() => {
+                            setFieldValue('phasedIn', true);
+                          }}
+                        />
+                        <Field
+                          as={Radio}
+                          id="phasedIn-No"
+                          name="phasedIn"
+                          label={h('no')}
+                          value="NO"
+                          checked={values.phasedIn === false}
+                          onChange={() => {
+                            setFieldValue('phasedIn', false);
+                          }}
+                        />
+                      </Fieldset>
+                    </FieldGroup>
+
+                    <div className="grid-col-12">
+                      <Button
+                        type="button"
+                        className="usa-button usa-button--unstyled margin-top-4"
+                        onClick={() => setHasAdditionalNote(!hasAdditionalNote)}
+                      >
+                        <IconAdd className="margin-right-1" aria-hidden />
+                        {h('additionalNote')}
+                      </Button>
+                    </div>
+
+                    {hasAdditionalNote && (
+                      <FieldGroup className="margin-top-4 grid-col-12">
+                        <Field
+                          as={TextAreaField}
+                          className="height-15"
+                          id="ModelType-phasedInNote"
+                          name="phasedInNote"
+                          label={t('Notes')}
+                        />
+                      </FieldGroup>
+                    )}
+
+                    <div className="margin-top-6 margin-bottom-3">
+                      <Button
+                        type="button"
+                        className="usa-button usa-button--outline margin-bottom-1"
+                        onClick={() =>
+                          history.push(
+                            `/models/${modelId}/task-list/basics/overview`
+                          )
+                        }
```

Comment:
I think for continuity the redirection should happen in the submit handler.  `Back` on milestones is handled differently that in overview.  `Back` should still save the form, so this should mimic the back functionality of overview